src/main/java/com/crud_read_and_create/controller/GameController.java

```diff
@@ -44,7 +45,9 @@ public String getSearch() {
 	}
 
 	@GetMapping("/create")
-	public String getCreate() {
+	public String getCreate(Model model) {
+		List<Platform> platformList = gameService.getPlatform();
+		model.addAttribute("platformList", platformList);
 		return "/create";
```

Comment:
htmlのファイル名を指定するのがいいですね。
```suggestion
		return "create";
```

Reply:
https://github.com/renangton/task5_crud_read_and_create/pull/15/commits/30808d0c5f5ebb9f7af02145d78c6df4e4bc6372
対応しました！